src/lib389/lib389/cli_conf/replication.py

```diff
@@ -1219,157 +1219,157 @@ def create_parser(subparsers):
-
-    repl_get_per_backend_cl = repl_subcommands.add_parser('get-changelog', help='Display replication changelog attributes.')
+    repl_set_per_backend_cl.add_argument('--suffix', required=True, help='Sets the suffix that uses the changelog')
+    repl_set_per_backend_cl.add_argument('--max-entries', help="Sets the maximum number of entries to get in the replication changelog")
+    repl_set_per_backend_cl.add_argument('--max-age', help="Set the maximum age of a replication changelog entry")
+    repl_set_per_backend_cl.add_argument('--trim-interval', help="Sets the interval to check if the replication changelog can be trimmed")
+    repl_set_per_backend_cl.add_argument('--encrypt', action='store_true', help="Sets the replication changelog to use encryption. You must export and import the changelog after setting this.")
+    repl_set_per_backend_cl.add_argument('--disable-encrypt', action='store_true', help="Sets the replication changelog to not use encryption. You must export and import the changelog after setting this.")
+
+    repl_get_per_backend_cl = repl_subcommands.add_parser('get-changelog', help='Display replication changelog attributes')
     repl_get_per_backend_cl.set_defaults(func=get_per_backend_cl)
-    repl_get_per_backend_cl.add_argument('--suffix', required=True, help='The suffix that uses the changelog')
+    repl_get_per_backend_cl.add_argument('--suffix', required=True, help='Sets the suffix that uses the changelog')
 
-    repl_export_cl = repl_subcommands.add_parser('export-changelog', help='Export the Directory Server replication changelog to an LDIF')
-    export_subcommands = repl_export_cl.add_subparsers(help='Export Replication Changelog')
-    repl_export_cl = export_subcommands.add_parser('to-ldif', help='Export the specific single LDIF file.  '
+    repl_export_cl = repl_subcommands.add_parser('export-changelog', help='Export the Directory Server replication changelog to an LDIF file')
+    export_subcommands = repl_export_cl.add_subparsers(help='Export replication changelog')
+    repl_export_cl = export_subcommands.add_parser('to-ldif', help='Sets the LDIF file name. '
                                                                    'This is typically used for setting up changelog encryption')
     repl_export_cl.set_defaults(func=dump_cl)
     repl_export_cl.add_argument('-c', '--csn-only', action='store_true',
-                                help="Export and interpret CSN only. This option can be used with or without -i option.  "
-                                     "The LDIF file that is generated can not be imported and is only used debugging purposes")
+                                help="Enables to export and interpret CSN only. This option can be used with or without -i option. "
+                                     "The LDIF file that is generated can not be imported and is only used debugging purposes.")
```

Comment:
Probably, it should be 'is only used **for** debugging purposes'

Reply:
fixed.